tools/sctool/src/main/java/com/horizen/CommandProcessor.java

```diff
@@ -431,6 +435,7 @@ private void processGenesisInfo(JsonNode json) {
             resJson.put("mcBlockHeight", mcBlockHeight);
             resJson.put("mcNetwork", mcNetworkName);
             resJson.put("withdrawalEpochLength", withdrawalEpochLength);
+            resJson.put("initialMcCumulativeCommTreeHash", BytesUtils.toHexString(initialMcCumulativeCommTreeHash));
```

Comment:
check `shouldUpdateConfig` logic, we should change `updateTemplateFile` logic as well.

Reply:
Done.